program/c/src/oracle/upd_aggregate.h

```diff
@@ -175,7 +175,7 @@ static inline bool upd_aggregate( pc_price_t *ptr, uint64_t slot, int64_t timest
            // No overflow for INT64_MIN+conf or INT64_MAX-conf as 0 < conf < INT64_MAX
            // These checks ensure that price - conf and price + conf do not overflow.
            (int64_t)0 < conf && (INT64_MIN + conf) <= price && price <= (INT64_MAX-conf) &&
-           slot_diff >= 0 && slot_diff <= PC_MAX_SEND_LATENCY ) {
```

Comment:
Was it voluntary to remove` slot_diff >= 0`

Reply:
I think we need to keep it to encourage a publisher sending a price in future slot and get that price included for all slots.

Reply:
this is redundant, there is a check slightly above to return false if `agg_.pub_slot_` is greater than current slot
```
  // only re-compute aggregate in next slot
  if ( slot <= ptr->agg_.pub_slot_ ) {
    return false;
  }
```

Reply:
ok